game/app/scenes/settings.gd

```diff
@@ -19,7 +24,10 @@ func _setup_language_selection():
 	languages.clear()
 	var active_language_code = TranslationServer.get_locale()
 	for language_code in TranslationServer.get_loaded_locales():
-		language_selection.add_item(TranslationServer.get_locale_name(language_code))
+		if LOCALE_NAMES.has(language_code):
+			language_selection.add_item(LOCALE_NAMES[language_code])
```

Comment:
Voluntary Extra Task Idea:
* If the locale name is defined, compare the result with the return value of `get_locale_name()`.
  * If they match, only set the locale name as the item label.
  * If they don't match, set the item label as `"%s (%s)" % [localized_name, engine_name]`. For English it would read `English`, while for German it would read `Deutsch (German)`.